include/Note.h

```diff
@@ -205,6 +205,12 @@ class LMMS_EXPORT Note : public SerializingObject
 		return classNodeName();
 	}
 
+	inline std::string getNoteName() const
```

Comment:
I'd rename the function to `getNoteString`, because down below you're using "name" to refer to just a letter and # without an octave number.